packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -144,7 +144,7 @@ impl DatabaseApi<'_> {
         &mut self,
         version_id: VersionId,
     ) -> Result<Persisted<DataType>, QueryError> {
-        self.store.get_data_type(version_id).await
+        self.store.get_data_type(&version_id).await
```

Comment:
This is what we discussed off-GitHub about not being able to pass by value?

Reply:
Yes 🙁